src/metabase/api/card.clj

```diff
@@ -468,4 +469,5 @@
   (db/select [Card :name :id], :enable_embedding true, :archived false))
 
 
-(api/define-routes)
+(api/define-routes
+  (middleware/streaming-json-response POST_:card-id_query))
```

Comment:
It's a little odd to use auto-generated names like this, kinda "pierces the vale" on the magic of these macros. ideas?